cronjobs_cern/indico_cronjobs_cern/plugin.py

```diff
@@ -10,16 +10,21 @@
 from indico.web.forms.base import IndicoForm
 from indico.web.forms.fields import EmailListField, MultipleItemsField
 
+from wtforms.fields import EmailField
+
 
 def _order_func(object_list):
     return sorted(object_list, key=lambda r: natural_sort_key(r[1].full_name))
 
 
 class SettingsForm(IndicoForm):
     _fieldsets = [
+        ('Sender', ['sender_email']),
         ('Seminar emails', ['seminar_categories', 'seminar_recipients'])
     ]
 
+    sender_email = EmailField ('Sender')
```

Comment:
```suggestion
    sender_email = EmailField('Sender', [Email()])
```

(also import that validator)

Reply:
Updated